tests/fxl-orientation_landscape/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-18</dc:date>
+    <dc:description>rendition:orientation is landscape, Reading Systems should either display the content in landscape or inform the user it should be.</dc:description>
```

Comment:
Nit: comma splice! Either use a period, colon, or semi-colon instead of the comma, or add a word like "so".

Reply:
And while we're at it, I prefer your wording on the pages themselves: "inform the user it should be viewed in landscape."